File: Learn/Algorithms/Warmup/Mini-Max Sum/minimax.py
import sys
# given
arr = list(map(int, input().strip().split(' ')))

# start
min_value = sys.maxsize
max_value = -(sys.maxsize - 1)
sum = 0

for i in arr:
    sum += i
    if i < min_value:
        min_value = i
    if i > max_value:
        max_value = i

print(sum - max_value, sum - min_value)

File: Learn/Algorithms/Warmup/Mini-Max Sum/minimax.c
#include <stdio.h>
#include <limits.h>

int main() {
    int min_value = INT_MAX;
    int max_value = INT_MIN;
    long int sum = 0;
    for (int i = 0; i < 5; i++) {
        int n;
        scanf("%d", &n);
        sum += n;
        if (n > max_value) {
            max_value = n;
        }
        if (n < min_value) {
            min_value = n;
        }
    }

    printf("%li %li\n", sum - max_value, sum - min_value);

    return 0;
}
